### PR TITLE
Add alias for PixelFEDChannelCollection.

### PR DESCRIPTION
**Pull request by tsusa**
This PR is the follow up of the PRs: #25466 and #25748 that introduced the simulation of the StuckTBMs. In the current implementation the code does not work properly, due to the fact that the same "InputLabel" is used in DigiToRaw for two collections [DetSetVector<PixelDigi> and PixelFEDChannelCollection]. In EventFilter/SiPixelRawToDigi/python/SiPixelDigiToRaw_cfi.py "InputLabel" is set to "simSiPixelDigis" and  there is no PixelFEDChannelCollection with that label, instead PixelFEDChannelCollection has the label "mix" or "mixData". In order to fix the problem, the same label "simSiPixelDigis" is also set for PixelFEDChannelCollection in digitizer
.

### CODE REVIEW

**Comment by tsusa** [comment]
@mmusich @tvami @leac @veszpv

**Comment by tsusa** [comment]
type: bug-fix

**Comment by leac** [comment]
> 
> 
> @mmusich @tvami @leac @veszpv

Why was I mentioned? :)

**Comment by tvami** [comment]
> > @mmusich @tvami @leac @veszpv
> 
> Why was I mentioned? :)

It was meant for her: https://gitlab.cern.ch/leac

**Comment by mmusich** [comment]
@leaca correct gitHub tag

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
+1

**Comment by makortel** [comment]
hold (even if I don't have powers)

I have some comments that I'll try to write later today.

**Comment by civanch** [comment]
hold

File: SimGeneral/MixingModule/python/aliases_cfi.py [hunk 1]
```diff
@@ -87,3 +87,13 @@
...
+
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+phase1Pixel.toModify(
+    simSiPixelDigis,
+    mix = cms.VPSet(
+        cms.PSet(type = cms.string('PixelDigiedmDetSetVector')),
+        cms.PSet(type = cms.string('PixelDigiSimLinkedmDetSetVector')),
+        cms.PSet(type = cms.string('PixelFEDChanneledmNewDetSetVector'))
+        )
+    )
```
[the remark this answers is not in the document]
Comment (makortel) [reply SimGeneral/MixingModule/python/aliases_cfi.py]: The repeating of the existing elements of `simSiPixelDigis.mix` could be avoided with something along
```python
phase1Pixel.toModify(simSiPixelDigis, mix = simSiPixelDigis.mix.values() + [cms.PSet(type = cms.string('PixelFEDChanneledmNewDetSetVector'))])
```

**Review by makortel — COMMENTED** [review]
Comments here (the fastsim-modifier one is the most relevant one, the two others are more aesthetics)

**Comment by fabiocos** [comment]
please test

**Comment by mmusich** [comment]
@makortel @civanch
can you please check again if this is now fine ? It would be very important for us to try to get this included in 10.6.0_pre1

**Comment by makortel** [comment]
@cmsbuild, please test workflow 25402.18,250402.181

**Comment by makortel** [comment]
Looks ok to me, but let's test the fastsim pileup workflows just to be sure.

File: SimGeneral/MixingModule/python/aliases_PreMix_cfi.py [hunk 2]
```diff
@@ -56,3 +56,13 @@
...
+
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+phase1Pixel.toModify(
+    simSiPixelDigis,
+    mix = cms.VPSet(
+        cms.PSet(type = cms.string('PixelDigiedmDetSetVector')),
+        cms.PSet(type = cms.string('PixelDigiSimLinkedmDetSetVector')),
+        cms.PSet(type = cms.string('PixelFEDChanneledmNewDetSetVector'))
+        )
+    )
```
[the remark this answers is not in the document]
Comment (tsusa) [reply SimGeneral/MixingModule/python/aliases_PreMix_cfi.py]: The change is kept in aliases_PreMix_cfi.py

File: SimGeneral/MixingModule/python/aliases_cfi.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (tsusa) [reply SimGeneral/MixingModule/python/aliases_cfi.py]: @matti corrected in the new commit

**Comment by fabiocos** [comment]
The extra tests seem to have been ignored by cms-bot, I will restart them as soon as the reference IB is ok

**Comment by tsusa** [comment]
We run  the tests:
runTheMatrix.py -l 25402.18,250402.181
Here is the result:
20:47 2019-date Thu Mar  7 13:12:10 2019; exit: 0 0 0
2 2 2 tests passed, 0 0 0 failed

**Comment by fabiocos** [comment]
@tsusa @makortel I can confirm I got the same in my standalone test

**Comment by civanch** [comment]
unhold

**Comment by civanch** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**